### ISSUE TITLE
Feature Request: Expose dbus-org.freedesktop.resolve1.service 

### ISSUE THREAD

**Issue by oniGino** [DNSCrypt/dnscrypt-proxy#1616]
## Is your feature request related to a problem?
I keep running into random apps and services that expect this service. Its usage is becoming more popular

## Describe the solution you'd like
By exposing at least partial support for dbus-org.freedesktop.resolve1.service
It allows dnscrypt-proxy to become a more feature complete replacement for systemd-resolved

I think we could just silently ignore many of the Set*() functions while answering  Resolve*() functions

## Alternatives you've considered
Today I just ignore the errors, its not critical, but I think it would be a nice addition

## Who will that feature be useful to?
Systemd users, modern major distro users

## What have you done already?
nada

## What are you going to contribute?
I can work on some code and try to build a PR.

**Comment by jedisct1** [comment]
Unfortunately I don't use Linux and I don't have any idea what that is and what it does :/

So your help would be very useful here!

**Comment by oniGino** [comment]
TLDR its a way for apps to request dns resolutions as well as set which dns servers are used and it happens via dbus

see the link below for the API spec
 https://manpages.ubuntu.com/manpages/groovy/man5/org.freedesktop.resolve1.5.html
